### DISCUSSION TITLE
wallet mnemonics 24 words is giving me a new address and not recognizing my plots

### DISCUSSION THREAD

**Discussion by pepperminttt** [Chia-Network/chia-blockchain#6343] [Support]
Hi
using chia 1.1.6
GUI
I just finished about 600 plots on 2 pcs the plots were working fine every time I fill a new hard drive I just connect it to the 2nd pc (main farmer) so when I had all my drives plotted, I deleted my key from the first pc and yes I know what you are thinking I do have a printed copy of my back up mnemonics 24 words.
On the farmer pc, I created a 2nd key I logged out from the main key logged in with the new key, it was all good.
so I deleted the old key and restarted chia with the new key but no plots recognized with the new key, so I imported my old key from my 24 words backup and I noticed the new public key number is different from the old one I had I logged in with that key but again no plots are recognized and I have 0.00000 in my wallet. 
I am new to chia I need asp help on this issue plz.

**Comment by room101-dev** [comment]
This post is BS, because noobs don't have 600 plots

But who cares, right?

First of all go back to the plot machines and run <. ./activate> **"chia keys show --mneumonic"** and do that for every machine ( that you ever made plots on ) and print the priv key, public key, and the funny little words that noobs love so much.

What matters is the private-key, and the public-key is in the plots you ( or somebody else created ), you can look for that public key

You say you deleted your wallet, what the hell does that mean? On the GUI, or did you go to ~/.chia/mainnet/wallet and delete those files?

The thing is the 24-word is only for recovery, they also use a deterministic-seed to generate addresses. You need to do what I wrote above and find out exactly what priv-key/pub-key pair is associated with your plots. If true that you deleted those files, then you are SOL.

U might be able recover using the seed-words, but I"m going to guess that all your plots probably had different keys, hell I don't know what you did, you don't know, and you haven't given us any info.

What I told you above will allow you to recover, if & only if, you didn't delete the wallet files.

If your decide to recover, I would start on a new machine, only run 'chia start wallet', and use 'cmd-line' tools like 'chia wallet show', 'chia wallet load 'file-name'', .... learn chia wallet --help

The GUI is shit-ware on CHIA, this 99% of all problems, just like BITCOIN if you can't use the cmd-line then you don't belong there.

Lastly, don't use windows, use linux, and never use the GUI.

**Comment by pepperminttt** [comment]
solved I was using the wrong mnemonics 24 words

**Comment by BrandtH22** [comment]
@pepperminttt , based on your previous message it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
